src/features/walletconnect/services/utils.ts

```diff
@@ -50,6 +50,10 @@ export const isBlockedBridge = (origin: string) => {
   return BlockedBridges.some((bridge) => origin.includes(bridge))
 }
 
+export const isSafePassApp = (origin: string) => {
+  return origin.includes('community.safe.global')
```

Comment:
Can we if not avoid hardcode then at least make it a constant?

Reply:
Can we use the constant `SAFE_PASS_URL` here?